Content.Server/StationEvents/Components/BureaucraticErrorRuleComponent.cs

```diff
@@ -1,9 +1,15 @@
-﻿using Content.Server.StationEvents.Events;
+using Content.Server.StationEvents.Events;
+using Content.Shared.Roles;
+using Robust.Shared.Prototypes;
 
 namespace Content.Server.StationEvents.Components;
 
 [RegisterComponent, Access(typeof(BureaucraticErrorRule))]
 public sealed partial class BureaucraticErrorRuleComponent : Component
 {
-
+    /// <summary>
+    /// The jobs that are ignored by this rule and won't have their slots changed.
+    /// </summary>
+    [DataField]
+    public HashSet<ProtoId<JobPrototype>> IgnoredJobs = new();
```

Comment:
this should just be a list

also ValidatePrototypeId is legacy, what you have here is fine for validation